en/micro-integrator/docs/develop/injecting-parameters.md

```diff
@@ -293,6 +296,9 @@ In the following example, JMS transport parameters in an inbound endpoint are co
 
 The <b>pinned servers</b> parameter as well as all the service-level <b>transport parameters</b> can be dynamically injected to a proxy service.
 
+!!! Note
+    The pinned server parameter is applicable only when in non-clustering mode.
```

Comment:
Also I guess the internal discussion is only about the scheduled tasks and not proxies. So if we are adding note to all the places where we have this parameter, we need to verify them first.